test/loader/adapter_registry/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: MIT
+
+set(DUMMY_LIB_DIR ${CMAKE_BINARY_DIR}/bin/${CMAKE_BUILD_TYPE}/adapter-reg)
+
+if(MSVC)
+    set(DUMMY_LIB_DIR ${CMAKE_BINARY_DIR}/bin/${CMAKE_BUILD_TYPE}/adapter-reg)
+else()
+    set(DUMMY_LIB_DIR ${CMAKE_BINARY_DIR}/bin/adapter-reg)
+endif()
+
+if(WIN32)
+    set(DUMMY_LIB_PATH ${DUMMY_LIB_DIR}/ur_adapter_level_zero.dll)
+else()
+    set(DUMMY_LIB_PATH ${DUMMY_LIB_DIR}/libur_adapter_level_zero.so.0)
+endif()
+
+# Create a dummy adapter library in adapter the runtime output directory
```

Comment:
in the adapter runtime output directory?

Reply:
I've removed this part.